py/dml/dmlc.py

```diff
@@ -233,6 +233,25 @@ def main(argv):
         '--dep', dest = "makedep", action = 'store',
         help = 'generate makefile dependencies')
 
+    # <dt>--no-dep-phony</dt>
+    # <dd>With --dep, avoid addition of a phony target for each dependency
+    # other than the main file.</dd>
+    optpar.add_option(
+        '--no-dep-phony', dest = "no_dep_phony", action = 'store_true',
+        help = 'With --dep, avoid addition of a phony target for each'
+        + ' dependency other than the main file.')
+
+    # <dt>--dep-target</dt>
+    # <dd>With --dep, change the target of the rule emitted by dependency
+    # generation. Specify multiple times to have multiple targets.</dd>
```

Comment:
I'd prefer if it were possible to specify multiple targets with only one specification of `--dep-target`. Is it?

Reply:
The user of this flag will be our CMake (or possibly some GNU Make?) code where it doesn't matter what we do here. I think having explicit --dep-target x --dep-target y --dep-target z is better and more explicit than --dep-target x y z

Also, --warn down below works exactly like this option, making it consistent.